src/Services/Cda/CdaTemplateImportDispose.php

```diff
@@ -621,7 +621,7 @@ public function InsertEncounter($enc_array, $pid, CarecoordinationTable $carecoo
                 } else {
                     //to lists
                     $query_insert = "INSERT INTO lists(pid,type,begdate,activity,title,date, diagnosis, subtype) VALUES (?,?,?,?,?,?,?,?)";
-                    $result = $appTable->zQuery($query_insert, array($pid, 'medical_problem', date("Y-m-d H:i:s", strtotime($value['encounter_diagnosis_date'])), 1,
+                    $result = $appTable->zQuery($query_insert, array($pid, 'medical_problem', $diag_date, 1,
```

Comment:
sorry, I set it up and somehow forgot it in insert. still, that date should never be empty and makes me concerned something happened in parse!